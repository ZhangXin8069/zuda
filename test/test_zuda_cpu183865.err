In file included from /public/home/zhangxin/external-libraries/numpy/core/include/numpy/ndarraytypes.h:1929,
                 from /public/home/zhangxin/external-libraries/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /public/home/zhangxin/external-libraries/numpy/core/include/numpy/arrayobject.h:5,
                 from zuda_cpu.cpp:1120:
/public/home/zhangxin/external-libraries/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
Traceback (most recent call last):
  File "/public/home/zhangxin/zuda/test/./test_zuda_cpu.py", line 13, in <module>
    init()
  File "/public/home/zhangxin/external-libraries/pyquda/mpi.py", line 55, in init
    cuda.Device(gpuid).use()
  File "cupy/cuda/device.pyx", line 192, in cupy.cuda.device.Device.use
  File "cupy/cuda/device.pyx", line 198, in cupy.cuda.device.Device.use
  File "cupy_backends/cuda/api/runtime.pyx", line 374, in cupy_backends.cuda.api.runtime.setDevice
  File "cupy_backends/cuda/api/runtime.pyx", line 143, in cupy_backends.cuda.api.runtime.check_status
cupy_backends.cuda.api.runtime.CUDARuntimeError: cudaErrorNoDevice: no CUDA-capable device is detected
