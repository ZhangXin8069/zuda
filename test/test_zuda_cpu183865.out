PATH:/public/home/zhangxin/zuda/test
~/zuda ~/zuda/test
HOME:/public/home/zhangxin/zuda
~/zuda/test
~/zuda/build ~/zuda/test
###test_zuda_cpu.sh is running...:2023-07-31-22-32-31###
Compiling zuda_cpu.pyx because it changed.
[1/1] Cythonizing zuda_cpu.pyx
running build_ext
building 'zuda_cpu' extension
creating build
creating build/temp.linux-x86_64-cpython-310
/public/soft/linux-centos7-x86_64/gcc-4.8.5/gcc-10.3.0-uoicdrf766usj4ma5wxqq4zaqgatfyy3/bin/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -fwrapv -O2 -Wall -fPIC -O2 -isystem /public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/include -fPIC -O2 -isystem /public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/include -fPIC -I/public/home/zhangxin/external-libraries/numpy/core/include -I/public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/include/python3.10 -c zuda_cpu.cpp -o build/temp.linux-x86_64-cpython-310/zuda_cpu.o
creating build/lib.linux-x86_64-cpython-310
/public/soft/linux-centos7-x86_64/gcc-4.8.5/gcc-10.3.0-uoicdrf766usj4ma5wxqq4zaqgatfyy3/bin/g++ -shared -Wl,-rpath,/public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/lib -Wl,-rpath-link,/public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/lib -L/public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/lib -Wl,-rpath,/public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/lib -Wl,-rpath-link,/public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/lib -L/public/soft/linux-centos7-x86_64/gcc-10.3.0/miniconda3-22.11.1-jrusysshr36o4fjbexlhrpljufoxhu4b/lib build/temp.linux-x86_64-cpython-310/zuda_cpu.o -o build/lib.linux-x86_64-cpython-310/zuda_cpu.cpython-310-x86_64-linux-gnu.so
copying build/lib.linux-x86_64-cpython-310/zuda_cpu.cpython-310-x86_64-linux-gnu.so -> 
mv ./zuda_cpu.cpython-310-x86_64-linux-gnu.so /public/home/zhangxin/zuda/lib
mv ./build/lib.linux-x86_64-cpython-310/zuda_cpu.cpython-310-x86_64-linux-gnu.so /public/home/zhangxin/zuda/lib
~/zuda/test ~/zuda/build ~/zuda/test
~/zuda/build ~/zuda/test
###test_zuda_cpu.sh is done......:2023-07-31-22-32-38###
~/zuda/test
